pkgs/by-name/ea/easyeda2kicad/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  python3Packages,
+  pythonPackages,
```

Comment:
You should probably refer to `fetchPypi` by directly including it here in the inputs, and remove `pythonPackages` which is you don't need (it's for python2).